src/dirigera/hub/hub.py

```diff
@@ -88,6 +90,17 @@ def get(self, route: str) -> Dict[str, Any]:
         response.raise_for_status()
         return response.json()
 
+    def post(self, route: str, data: Dict[str, Any] = None) -> Dict[str, Any]:
+        response = requests.post(
+            f"{self.api_base_url}{route}",
+            headers=self.headers(),
+            json=data,
+            timeout=10,
+            verify=False,
+        )
+        response.raise_for_status()
+        return response.text
```

Comment:
```suggestion
        return response.json()
```
without this the return type is wrong

Reply:
The api endpoint /scenes/<id>/trigger returns a empty body so performing response.json() will raise an exception, I changed return type hint to Optional[Dict[str, Any]] and returns None if response body is empty.

I would propose making the same changes to Hub.patch (the return value does seem to be used but right now the type hint is Dict[str, Any] but the actual return value is response.text). Although I am not aware of any POST or PATCH endpoint that returns a body so the return value might as well be removed.

Reply:
Good point I will change this